src/test/java/edu/ucsb/mapache/controllers/SlackSlashCommandControllerTests.java

```diff
@@ -246,6 +254,17 @@ public void test_googleSearch_placeholder_placeholder() throws Exception {
                 .andExpect(status().is(200));
     }
 
+    @Test
+    public void test_whoisCommand() throws Exception {
```

Comment:
It would be nice to have a slightly more descriptive test name here, for documentation purposes, in case someone decides to add onto this method in the future and needs to add more tests. For example, `test_whoisCommand_correct_response` or somethings (just to indicate the specific case you're covering).

Reply:
I updated the method name and I am ready for a review again.